### PR TITLE
Remove unused ReportsController#user_stats

### PR DESCRIPTION
**Pull request by joshlory**
This removes the unused `/stats/:user_id` route, action and view.

### CODE REVIEW

File: dashboard/test/controllers/scripts_controller_test.rb [hunk 1]
```diff
@@ -230,4 +230,14 @@ def create_admin_script
     get :show, id: admin_script.name
     assert_response :success
   end
+
+  test "should have one game group if one stage" do
```
Comment (laurelfan) [inline dashboard/test/controllers/scripts_controller_test.rb:234]: this seems like it goes with something else? But if it passes feel free to check it in here :)

**Comment by laurelfan** [comment]
LGTM

File: dashboard/test/controllers/scripts_controller_test.rb (hunk 1 above)
Comment (joshlory) [inline dashboard/test/controllers/scripts_controller_test.rb:234]: Moved from [`reports_controller_test.rb`](https://github.com/code-dot-org/code-dot-org/pull/6314/files#diff-a688b3ce6b42334e911bbb3c2178a797L75).

Comment (laurelfan) [inline dashboard/test/controllers/scripts_controller_test.rb:234]: oh I see. does it also apply to header_stats like "should have two game groups if two stages" (and does that one also apply to scripts/show)?

**Pull request merged**